runtime/hsa-runtime/core/driver/kfd/amd_kfd_driver.cpp

```diff
@@ -42,34 +42,80 @@
 
 #include "core/inc/amd_kfd_driver.h"
 
-#include <sys/ioctl.h>
-
 #include <memory>
 #include <string>
 
+#include <link.h>
+#include <sys/ioctl.h>
+
 #include "hsakmt/hsakmt.h"
 
-#include "core/inc/amd_cpu_agent.h"
-#include "core/inc/amd_gpu_agent.h"
 #include "core/inc/amd_memory_region.h"
-#include "core/inc/exceptions.h"
 #include "core/inc/runtime.h"
 
+extern r_debug _amdgpu_r_debug;
```

Comment:
I think this may conflict with:
https://github.com/ROCm/ROCR-Runtime/commit/147abb6ca0b99859d97ee24518eaa344e060e959

But that's ok. We can try to pull in the changes as is and I can fix later.